.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint and Test Charts
```

Comment:
should we call this as "chart-lint-test.yml" and name as : "Chart Lint and Test" - to not confuse with go code lint and test.